roles/grubcmdline/tasks/main.yml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
Why remove this?

Reply:
Again, my ansible linter seems to have automatically removed that. I'll fix these two up